multiapp.py

```diff
@@ -39,8 +39,8 @@ def add_app(self, title, func):
         })
 
     def run(self):
-        # app = st.sidebar.radio(
-        app = st.selectbox(
+        app = st.sidebar.radio(
+        # app = st.selectbox(
```

Comment:
Remove the commented code.  With git we can always go see what was there before.  We always want to remove commented code to avoid confusion and developer cycles when they have to look at it.   Remove the line and then git push